.github/workflows/react-build.yml

```diff
@@ -0,0 +1,39 @@
+# This workflow will do a clean installation of node dependencies, cache/restore them, build the source code and run tests across different versions of node
+# For more information see: https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-nodejs
+
+name: React.js Build CI
+
+on:
+  push:
+    branches-ignore:
+      - main
+  pull_request:
+    # The branches below must be a subset of the branches above
+    branches:
+      - main
+
+jobs:
+  build:
+    defaults:
+      run:
+        working-directory: gui
```

Comment:
@fmaddenflx, did this end up being necessary with the inclusion of the directory in the path to the lock file?

Reply:
Not totally sure I understand your question.  Line 19 was necessary in order for the action to find the lock file.